src/editors/components/EditorFooter/index.jsx

```diff
@@ -18,8 +19,15 @@ import { selectors, thunkActions } from '../../data/redux';
 import messages from './messages';
 import * as module from '.';
 
-export const handleSaveClicked = (props) => () => saveBlock(props);
-export const handleCancelClicked = ({ returnUrl }) => navigateCallback(returnUrl);
+export const handleSaveClicked = (props) => () => {
+  sendTrackEvent('edx.ui.authoring.editor.save', {});
+  saveBlock(props);
+};
+
+export const handleCancelClicked = ({ returnUrl }) => {
+  sendTrackEvent('edx.ui.authoring.editor.cancel', {});
```

Comment:
So one thing we have been doing to avoid magic strings like this is having a javascript object stored in utils which takes as values the various ids of tracking events. Where this breaks down is that we want to have tracking events in the MFE and in this package, and passing them down into the editor as props seems unnecessary becuase we don't do the same for formatted message ids. I am wonder what you thoughts might be here @muselesscreator

Reply:
Value exists in simply having the event name listed at the exact place it's emitted. Also, this pattern seems to be a standard used elsewhere: https://github.com/openedx/frontend-app-learning/search?q=sendTrackEvent